test/kubemark/start-kubemark.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script that creates a Kubemark cluster with Master running on GCE.
```

Comment:
Actually, now that I think about it, I'm confused how this PR maps to the design doc. There doesn't seem to be any HollowNodes here. I do see that you use an RC to start multiple replicas of Kubelet, each in a pod. But IIUC there's just a single set of master components (hollow-cluster-master), as in Option 1? Where do the real nodes (that the hollow kubelet pods get run on) get created?

Actually, maybe I'm misunderstanding, but I think this is a hybrid between Option 1 and Option 2, right? That is, you use a single master (like Option 1) but you run the hollow nodes on top of real Kubernetes nodes (like Option 2)? If that's the case, then I think either Option 1 or Option 2 would actually be better, because in both of those options, the "master under test" is only serving hollow nodes, whereas here the "master under test" is serving hollow nodes and real nodes, so there will be performance interference. I admit I didn't see where the real nodes get started here (obviously the hollow nodes are started by the hollow-kubelet.json).